package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@wagtail/stylelint-config-wagtail",
   "version": "0.6.0",
-  "description": "Shareable stylelint config for CSS and SCSS, following Wagtail’s code style",
+  "description": "Shareable Stylelint config for CSS and SCSS, following Wagtail’s code style",
```

Comment:
Interesting, I thought `stylelint` was always written lowercase. Either I dreamt that or they changed it!

Reply:
I assumed it's a proper noun, their logo seems to be styled as lowercase but in their own readme copy they use it as a propr noun. 

> We recommend using a pretty printer like Prettier alongside Stylelint. Linters and pretty printers are complementary tools that work together to help you write consistent and error-free code.